portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,76 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@font-face {
+  font-family: Muli-Medium;
+  src: url("./assets/font/Muli-Medium.ttf") format("truetype");
 }
 
-#greeting-container {
+@font-face {
```

Comment:
You're not contributing to google3 codebase, so feel free to ignore my comment :)

Some general suggestions for writing css inside Google:
1) Keep the attributes sorted.
2) Avoid using element selector like body, h1. Try to replace them with id or class selectors.
3) Try to mark variable like font, color as a sharable const variable in a separate file. This can make it easy for your website to share a consistent UI style.

Reply:
Done